### ISSUE TITLE
Время сервера

### ISSUE THREAD

**Issue by Tomb-Raider** [Dostoyewski/PN_Expert#6]
Время жалоб пишет на минус три часа. Проверить в чём баг

**Comment by Tomb-Raider** [comment]
сервер сбрасывает все задания в 12 ночи, а выдаёт их по времени регистрации. Необходимо исправить. Лучше если задания всегда будут назначаться в 8 утра по часовому поясу. Если задание не выполнено, оно должно быть перенесено на следующий день с пометкой 'просрочено' @Dostoyewski

**Comment by Dostoyewski** [comment]
@Tomb-Raider Назначение заданий в 8 утра по часовому поясу сделать почти невозможно. Проще сделать раздел с "просрочкой".

**Comment by Dostoyewski** [comment]
Короче, сейчас вкину в модель задачи поле "просрочено". Вернее, попытаюсь его вкинуть.

**Comment by Dostoyewski** [comment]
Хотя валидацию можно сделать на стороне фронта. @lunakoly, что скажешь?

**Comment by Dostoyewski** [comment]
Короче, теперь возвращаются все ивенты. Проверку на стороне бэка делать нереально, проще это на стороне фронта сделать.

**Comment by Dostoyewski** [comment]
@Tomb-Raider продумай, насколько реально, чтобы пользователь при регитсрации выбирал свой часовой пояс. Так как бэк не контактирует с пользователем напрямую, получить его часовой пояс просто так я не могу.

**Comment by Dostoyewski** [comment]
@lunakoly насколько реально дергать часовой пояс пользователя при первом входе и отправлять его мне?

**Comment by Tomb-Raider** [comment]
@Dostoyewski да, можно сделать доп поле при регистрации, главное в бэк его добавь

**Comment by lunakoly** [comment]
@Tomb-Raider, нет, это поле должно быть внутри страницы настроек

**Comment by lunakoly** [comment]
@Dostoyewski, я запутался, какие в итоге события мне будут приходить в `events/current`?

**Comment by Dostoyewski** [comment]
> @Dostoyewski, я запутался, какие в итоге события мне будут приходить в `events/current`?

Все невыполненные.
